lib/searchable_by/column.rb

```diff
@@ -1,16 +1,19 @@
 module SearchableBy
   class Column
-    attr_reader :attr, :type, :match, :match_phrase
+    attr_reader :attr, :type, :match, :match_phrase, :min_length
     attr_accessor :node
 
-    def initialize(attr, type: :string, match: :all, match_phrase: nil)
+    def initialize(attr, type: :string, match: :all, match_phrase: nil, min_length: nil)
       @attr  = attr
       @type  = type.to_sym
       @match = match
       @match_phrase = match_phrase || match
+      @min_length = min_length
     end
 
     def build_condition(value)
+      return Arel::Nodes::False.new if min_length && value.term.length < min_length # no-match
```

Comment:
I wonder if we instead have a pre-check method, something like `applicable?` and avoid creating an AST node altogether?

Reply:
@dim I decided to do `WHERE FALSE` for such case to return no results (for this column). If I'd just skip it, it would behave incorrectly (basically, would return all recs if this would be the only condition).

So I think we must have an Arel node for it. We could have `applicable?`, but that would be pretty much the same thing in the end, just done in another place.

Makes sense? Have I missed anything?